Spanner/src/Operation.php

```diff
@@ -261,11 +281,13 @@ public function executeUpdate(
         Session $session,
         Transaction $transaction,
         $sql,
-        array $options = []
+        array $options = [],
+        &$transactionId = null
```

Comment:
Can we get the transactionId from `$transaction` instead of adding additional param here? I see we already have a function `getIdReference` for that purpose.

Reply:
I can do that, just added this variable for uniformity with other functions.

Reply:
This logic is changed now, hence resolving the comment. Also, the transaction object in the new logic is being passed in the options array. Thank you for the suggestion and it was helpful.